nmodl/__init__.py

```diff
@@ -1,7 +1,25 @@
+import os
+import sys
+
+if sys.version_info >= (3, 9):
+    from importlib.resources import files
+else:
+    from importlib_resources import files
+
+from find_libpython import find_libpython
+
+# add libpython*.so path to environment
+os.environ["NMODL_PYLIB"] = find_libpython()
```

Comment:
Is setting these unconditionally desired? Should we allow a use to pick the value of those variables?

Reply:
`os.environ["NMODL_PYLIB"] = os.environ.get("NMODL_PYLIB", find_libpython())` would admittedly be more flexible. Note that `find_libpython` either returns a string or `None` instead of raising an or returning an empty string, so maybe wrapping the above in a `try..except TypeError` block and raising a `RuntimeError`?

Reply:
Build system related bugs are horrible to debug. Therefore, yes, that might save someone some time.

Reply:
Updated the env mechanism now